package.json

```
@@ -12,17 +12,19 @@
         "@mui/icons-material": "^5.5.1",
         "@mui/lab": "^5.0.0-alpha.75",
```

not the last version
```suggestion
        "@mui/lab": "^5.0.0-alpha.168",
```

---

linked to comment [behind](https://github.com/gridsuite/gridstudy-app/pull/1997#discussion_r1525093486)